AK/Math.h

```diff
@@ -72,6 +179,20 @@ constexpr T fmod(T x, T y)
         : "0"(x), "u"(y));
     return res;
 }
+template<FloatingPoint T>
+ALWAYS_INLINE T modf(T val, T& int_part)
+{
+    // FIXME: CONSTEXPR_STATE
```

Comment:
this has no meaning to me. What is this comment saying?

Reply:
as you can see modf is currently not marked as constexpr and does not use the helper for the evaluation at compile time,
although if I look closely, it could be constexpr, due to not directly using assembly.